src/components/Modal/Box.tsx

```diff
@@ -20,21 +20,45 @@ interface MergedStyledProps extends InjectedProps {
   left?: number
 }
 
-const BoxComponent: React.FC<Props & InjectedProps> = ({
-  active,
-  children,
-  hideModal,
-  ...props
-}) => (
-  <Wrapper className={active ? 'active' : ''} {...props}>
-    {active ? (
-      <React.Fragment>
-        <Background {...props} onClick={hideModal} />
-        <Inner {...props}>{children}</Inner>
-      </React.Fragment>
-    ) : null}
-  </Wrapper>
-)
+interface State {
+  scrollTop: number
```

Comment:
I'm not sure why the `scrollTop` is required as a state.
Do you have any problems if you only apply `overflow: hidden` to the body without managing `scrollTop`?

Reply:
If implemented with `overflow: hidden` only, it worked fine.
It seems that I was thinking too hard... 😭